rfcs/convergence/props-consistency-check.md

```diff
@@ -0,0 +1,391 @@
+# Consistency Check
+
+The following components and their props were reviewed as of 9/7/2021 for consistency.
+
+Accordian, AccordianHeader, AccordianItem, AccordianPanel
+Avatar
+Badge, CounterBadge, PresenceBadge,
+Button, CompoundButton, MenuButton, ToggleButton,
+Divider,
+Image,
+Label,
+Link,Menu, MenuDivider, MenuGroup, MenuGroupHeader,MenuItem, MenuItemCheckbox, MenuItemRadio, MenuList, MenuPopover, MenuTrigger
+Popover, PopoverSurface, PopoverTrigger
+Portal
+
+## Nit: Explicit component type declaration
+
+> Some components declare their type explicity, others do not.
```

Comment:
From @ecraig12345 in HackMD:
At least IMO it's strongly preferable to have explicit type declarations, and this is one of the things I'd like to write a conformance test for. (Also the & React.RefAttributes part isn't needed because it's included in ComponentProps somewhere.) It's a cleaner API from a consumer standpoint and sometimes shows up MUCH cleaner in API reports by preventing unnecessary type expansion.

It can be simplified to: `export const Accordion: React.FunctionComponent<AccordionProps> = React.forwardRef((props, ref) => ...)`

Reply:
from Guest Rodriques in HackMD:
https://github.com/microsoft/fluentui/pull/19614#discussion_r706000532

Reply:
from Guest Weaver in HackMD:
Please use at least `ForwardRefExoticComponent`